Assignment 3-JS/script/script.js

```diff
@@ -0,0 +1,312 @@
+  //to render years
+  function Years(){
+    let d = new Date();
+    let currentyear = d.getFullYear();
+    
+    for(let i = 2050;i >= 1950; i--) {
+      var yearsDiv = document.createElement("div");
+      yearsDiv.classList.add('year-name');
+      var text = document.createTextNode(i);
+      yearsDiv.appendChild(text);
+      var yearsContainer = document.getElementById("years");
+      yearsContainer.appendChild(yearsDiv);
+      if(i == currentyear) {
+        yearsDiv.classList.add('current-year');
+        let yearElement = document.querySelector('.current-year');
+        yearElement.scrollIntoView();
+        yearElement.classList.add('firstElement-to-scroll');
+        yearElement.style.color = " #00e6b8 ";
+      }
+    }
+  }
+  //to hide the side-panel
+  function closeNav() {
+    document.getElementById("myScrollpanel").style.height = "0";
+  }

```

Comment:
Indentation is not proper and add a line break.

Reply:
Done.